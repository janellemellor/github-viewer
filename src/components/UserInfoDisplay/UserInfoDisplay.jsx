import React from 'react';
import PropTypes from 'prop-types';

const UserInfoDisplay = ({ username, followerCount, followingCount, profileUrl }) => (
  <>
    <p>{username}</p>
    <p>{followerCount}</p>
    <p>{followingCount}</p>
    <p>{profileUrl}</p>
  </>
);

UserInfoDisplay.propTypes = {
  username: PropTypes.string.isRequired,
  followerCount: PropTypes.string.isRequired,
  followingCount: PropTypes.string.isRequired,
  profileUrl: PropTypes.string.isRequired
};

export default UserInfoDisplay;
