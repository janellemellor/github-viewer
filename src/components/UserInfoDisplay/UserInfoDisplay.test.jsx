import React from 'react';
import { shallow } from 'enzyme';
import UserInfoDisplay from './UserInfoDisplay';

describe('UserInfoDisplay', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<UserInfoDisplay username='janellemellor'
      followerCount='2'
      followingCount='1'
      profileUrl='https://github.com/janellemellor' />);
    expect(wrapper).toMatchSnapshot();
  });
});

