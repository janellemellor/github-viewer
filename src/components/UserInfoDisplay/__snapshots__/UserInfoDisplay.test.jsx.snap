// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfoDisplay matches a snapshot 1`] = `
<Fragment>
  <p>
    janellemellor
  </p>
  <p>
    2
  </p>
  <p>
    1
  </p>
  <p>
    https://github.com/janellemellor
  </p>
</Fragment>
`;
